.github/ISSUE_TEMPLATE/joint-security-review.yaml

```diff
@@ -0,0 +1,103 @@
+name: 'Joint Security Review'
+description: To request a joint review or track progress on an active review
+title: "[Security Review]: "
+labels:
+  - tag/security-and-compliance
+  - needs-triage
+body:
+  - type: input
+    id: project-name
+    attributes:
+      label: Project Name
+      description: The name of the project to be reviewed.
+      placeholder: e.g., My CNCF Project
+    validations:
+      required: true
+  - type: input
+    id: github-url
+    attributes:
+      label: GitHub URL
+      description: The main GitHub repository URL for the project.
+      placeholder: e.g., https://github.com/org/project-name
+    validations:
+      required: true
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Project Security Contacts
+      description: |
+        List the GitHub usernames of the security representatives from the project that
+        will particpate in the review
+      placeholder: |
+        @contact1
+        @contact2
```

Comment:
Does this get filled in under "project security lead" in the check box portion below?  We may only need this one place/